### PR TITLE
feat: generate HTTP body using json-schema-faker

### PR DESCRIPTION
**Pull request by formulahendry**
Repair API:
image

### CODE REVIEW

File: src/commands/generateHttpFile.ts [hunk 1]
```diff
@@ -118,12 +116,22 @@ function parseBody(requestBody: OpenAPIV3.RequestBodyObject | undefined): string
         let body: { [key: string]: any } = {};
         if (jsonBodySchema.example) {
             body = jsonBodySchema.example;
+        } else if (jsonBodySchema.default) {
```
Comment (wenytang-ms) [inline src/commands/generateHttpFile.ts:119]: why example prior to default?

File: src/commands/generateHttpFile.ts [hunk 2]
```diff
@@ -118,12 +116,22 @@ function parseBody(requestBody: OpenAPIV3.RequestBodyObject | undefined): string
...
+            body = jsonBodySchema.default;
         } else if (jsonBodySchema.properties) {
             for (const name in jsonBodySchema.properties) {
                 const propertySchema = jsonBodySchema.properties[name] as OpenAPIV3.SchemaObject;
-                body[name] = propertySchema.example;
+                if (propertySchema.example) {
+                    body[name] = propertySchema.example;
+                }
+                if (propertySchema.default) {
+                    body[name] = propertySchema.default;
```
Comment (wenytang-ms) [inline src/commands/generateHttpFile.ts:128]: will default overwrite example?

File: src/commands/generateHttpFile.ts (hunk 1 above)
Comment (formulahendry) [reply src/commands/generateHttpFile.ts:119]: An example illustrates what the value is supposed to be. A default value is what the server uses if the client does not provide the value.
Refer to https://swagger.io/docs/specification/adding-examples/#:~:text=An%20example%20illustrates%20what%20the,does%20not%20provide%20the%20value.

The `example` would be a little more real. So use `example` first.

File: src/commands/generateHttpFile.ts (hunk 2 above)
Comment (formulahendry) [reply src/commands/generateHttpFile.ts:128]: Good catch!

File: package.json [hunk 3]
```diff
@@ -345,6 +345,7 @@
     "express": "^4.19.2",
     "fs-extra": "^11.1.1",
     "get-port": "^7.0.0",
+    "json-schema-faker": "^0.5.6",
```
Comment (adashen) [inline package.json:348]: since new dependency added. Please check the CG to make sure there is no alert. and may be update thiredparty notice.

Comment (formulahendry) [reply package.json:348]: No alert:


And updated thiredparty notice.

**Review by adashen — APPROVED** [review]

**Pull request merged**